analyses/molecular-subtyping-HGG/run-molecular-subtyping-HGG.sh

```diff
@@ -14,6 +14,12 @@ set -o pipefail
 # files gets run -- it will be turned off in CI
 SUBSET=${OPENPBTA_SUBSET:-1}
 
+# This controls whether or not notebooks that explore and are not used directly
+# for subtyping are run. By default (when this is set to 1), these notebooks
+# will not be run, i.e., subtyping notebooks only will be run.
+# It is not intended to be used in CI.
+SUBSETTING_ONLY=${SUBSETTING_ONLY:-1}
```

Comment:
Where is this setting likely to be used? For example I would have expected to see it in [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/cc2879c25c12fe2074a3f13764d6bdb057bd9901/scripts/run-for-subtyping.sh).

Reply:
This setting would be used if someone was interested in running all notebooks in the module, and I expect it be used less often than when folks just run this for subtyping. So by default, there are two notebooks we're skipping when this is run for subtyping; you would not expect to see it in [this script](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/cc2879c25c12fe2074a3f13764d6bdb057bd9901/scripts/run-for-subtyping.sh) because we're using the default settings.